### PR TITLE
Removing defunct service, translating names to Spanish and adding Buenos Aires City orthophotos

### PR DESCRIPTION
**Pull request by Zalitoar**
This PR contains the commits from the old #1425 PR, adding few modifications.

### CODE REVIEW

File: sources/south-america/ar/GCBA-ortofotos-2017.geojson [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        "url": "http://servicios.usig.buenosaires.gob.ar/mapcache/tms/1.0.0/fotografias_aereas_2017_caba_3857@GoogleMapsCompatible/{zoom}/{x}/{-y}.png",
+        "country_code": "AR",
+        "attribution": {
+            "text": "Gobierno de la Ciudad de Buenos Aires",
+            "url": "https://data.buenosaires.gob.ar/dataset/tiles-fotografia-aerea"
+        },
+        "icon": "https://www.buenosaires.gob.ar/sites/gcaba/files/favicon.png",
+        "max_zoom": 18,
+        "min_zoom": 9,
+        "license_url": "https://wiki.openstreetmap.org/wiki/File:Solicitud-de-permiso-para-usar-datos-de-BA-Data-en-OpenStreetMap.png",
```
Comment (matheusgomesms) [inline sources/south-america/ar/GCBA-ortofotos-2017.geojson:21]:
```suggestion
        "permission_osm": "explicit",
        "license_url": "https://wiki.openstreetmap.org/wiki/File:Solicitud-de-permiso-para-usar-datos-de-BA-Data-en-OpenStreetMap.png",
```

File: sources/south-america/ar/GCBA-ortofotos-2017.geojson [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        "description": "Fotografía aérea tomada en el año 2017",
+        "url": "http://servicios.usig.buenosaires.gob.ar/mapcache/tms/1.0.0/fotografias_aereas_2017_caba_3857@GoogleMapsCompatible/{zoom}/{x}/{-y}.png",
+        "country_code": "AR",
+        "attribution": {
+            "text": "Gobierno de la Ciudad de Buenos Aires",
+            "url": "https://data.buenosaires.gob.ar/dataset/tiles-fotografia-aerea"
+        },
+        "icon": "https://www.buenosaires.gob.ar/sites/gcaba/files/favicon.png",
+        "max_zoom": 18,
+        "min_zoom": 9,
```
Comment (matheusgomesms) [inline sources/south-america/ar/GCBA-ortofotos-2017.geojson:19]:
```suggestion
        "available_projections": [
            "EPSG:3857"
        ],
        "min_zoom": 9,
```

File: sources/south-america/ar/Educational-Map_wms.geojson [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "type": "Feature",
     "properties": {
         "id": "Mapa-Educativo-wms",
-        "name": "National Education Map (WMS)",
+        "name": "Mapa Educativo (WMS)",
```
Comment (matheusgomesms) [inline sources/south-america/ar/Educational-Map_wms.geojson]:
```suggestion
        "name": "Mapa Educativo",
```

File: sources/south-america/ar/IGN-UAV-orthophotos_wms.geojson [hunk 4]
```diff
@@ -2,7 +2,7 @@
     "type": "Feature",
     "properties": {
         "id": "ign-uav-orthophotos-wms",
-        "name": "National Geographic Institute UAV Orthophotos (WMS)",
+        "name": "IGN ortofotos VANT (WMS)",
```
Comment (matheusgomesms) [inline sources/south-america/ar/IGN-UAV-orthophotos_wms.geojson]:
```suggestion
        "name": "IGN ortofotos VANT",
```

File: sources/south-america/ar/IGN-orthophotos_wms.geojson [hunk 5]
```diff
@@ -2,7 +2,7 @@
     "type": "Feature",
     "properties": {
         "id": "ign-orthophotos-wms",
-        "name": "National Geographic Institute Orthophotos (WMS)",
+        "name": "IGN ortofotos (WMS)",
```
Comment (matheusgomesms) [inline sources/south-america/ar/IGN-orthophotos_wms.geojson]:
```suggestion
        "name": "IGN ortofotos",
```

File: sources/south-america/ar/IGN-vector_wms.geojson [hunk 6]
```diff
@@ -2,7 +2,7 @@
     "type": "Feature",
     "properties": {
         "id": "ign-wms",
-        "name": "IGN (WMS)",
+        "name": "IGN capas vectoriales (WMS)",
```
Comment (matheusgomesms) [inline sources/south-america/ar/IGN-vector_wms.geojson]:
```suggestion
        "name": "IGN capas vectoriales",
```

File: sources/south-america/ar/Secretary-of-Energy_wms.geojson [hunk 7]
```diff
@@ -2,7 +2,7 @@
     "type": "Feature",
     "properties": {
         "id": "Secretaria-de-Energia-wms",
-        "name": "Secretary of Energy (WMS)",
+        "name": "Secretaría de Energía (WMS)",
```
Comment (matheusgomesms) [inline sources/south-america/ar/Secretary-of-Energy_wms.geojson]:
```suggestion
        "name": "Secretaría de Energía",
```

**Comment by matheusgomesms** [comment]
Thanks a lot for your work and patience, @Zalitoar! I've reviewed your PR and looks good to me. I made some suggestions, so feel free to accept if you agree:

1 - In the new layer, I added the items _available_projections_ and _permission_osm_
2 - I removed the (WMS) chars from all layers. Does it make sense to keep them?

Once I receive your comments, I'll gladly merge this PR. Feliz año nuevo!

**Comment by Zalitoar** [comment]
Thanks @matheusgomesms I'm OK with your changes, let's move on this PR :)

**Comment by matheusgomesms** [comment]
> Thanks @matheusgomesms I'm OK with your changes, let's move on this PR :)

Good, can you (and do you know how to) accept the changes? I'm not sure how to do it, it seems blocked for me to edit it. After you accept them, I'll merge it. :)

**Comment by Zalitoar** [comment]
PR Check fails because it tries to get some map tiles in the middle of the ocean, probably because the layers bounding box includes both part of Antarctica and continental Argentina, so the centroid will be in the middle of the sea where there is no cache.

**Comment by matheusgomesms** [comment]
No problem, but the GeoJSON is not following the right-hand rule, so I'm fixing it now. Please accept that again, and let's finally merge it!

File: sources/south-america/ar/GCBA-ortofotos-2017.geojson [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+                [-58.53021, -34.63651],
+                [-58.53085, -34.63465],
+                [-58.53041, -34.63299],
+                [-58.53142, -34.61545],
+                [-58.50084, -34.5499],
+                [-58.4761, -34.5383],
+                [-58.46693, -34.53468]
+            ]
+        ]
+    }
```
Comment (matheusgomesms) [inline sources/south-america/ar/GCBA-ortofotos-2017.geojson]:
```suggestion
                            [
              -58.46693,
              -34.53468
            ],
            [
              -58.4761,
              -34.5383
            ],
            [
              -58.50084,
              -34.5499
            ],
            [
              -58.53142,
              -34.61545
            ],
            [
              -58.53041,
              -34.63299
            ],
            [
              -58.53085,
              -34.63465
            ],
            [
              -58.53021,
              -34.63651
            ],
            [
              -58.52872,
              -34.65476
            ],
            [
              -58.48878,
              -34.68491
            ],
            [
              -58.4866,
              -34.68821
            ],
            [
              -58.48211,
              -34.68978
            ],
            [
              -58.46141,
              -34.70563
            ],
            [
              -58.42735,
              -34.66546
            ],
            [
              -58.42369,
              -34.6623
            ],
            [
              -58.41667,
              -34.65959
            ],
            [
              -58.41267,
              -34.65878
            ],
            [
              -58.40694,
              -34.6603
            ],
            [
              -58.40254,
              -34.65979
            ],
            [
              -58.39743,
              -34.66218
            ],
            [
              -58.39253,
              -34.6626
            ],
            [
              -58.38775,
              -34.66059
            ],
            [
              -58.38453,
              -34.65798
            ],
            [
              -58.37386,
              -34.65715
            ],
            [
              -58.36944,
              -34.65387
            ],
            [
              -58.36846,
              -34.65117
            ],
            [
              -58.36399,
              -34.64886
            ],
            [
              -58.3618,
              -34.6484
            ],
            [
              -58.35655,
              -34.64487
            ],
            [
              -58.35759,
              -34.63989
            ],
            [
              -58.34901,
              -34.63249
            ],
            [
              -58.34608,
              -34.63165
            ],
            [
              -58.34793,
              -34.63116
            ],
            [
              -58.34879,
              -34.62907
            ],
            [
              -58.34706,
              -34.62725
            ],
            [
              -58.34536,
              -34.62867
            ],
            [
              -58.34457,
              -34.628
            ],
            [
              -58.34284,
              -34.62943
            ],
            [
              -58.34139,
              -34.6289
            ],
            [
              -58.34426,
              -34.62638
            ],
            [
              -58.34242,
              -34.62617
            ],
            [
              -58.33913,
              -34.62879
            ],
            [
              -58.33624,
              -34.62803
            ],
            [
              -58.33514,
              -34.62643
            ],
            [
              -58.34003,
              -34.62324
            ],
            [
              -58.34238,
              -34.62334
            ],
            [
              -58.34305,
              -34.62266
            ],
            [
              -58.33698,
              -34.62234
            ],
            [
              -58.33837,
              -34.61869
            ],
            [
              -58.34391,
              -34.61888
            ],
            [
              -58.34432,
              -34.6203
            ],
            [
              -58.34367,
              -34.62242
            ],
            [
              -58.3442,
              -34.6225
            ],
            [
              -58.34432,
              -34.6215
            ],
            [
              -58.34528,
              -34.6214
            ],
            [
              -58.34566,
              -34.62227
            ],
            [
              -58.34716,
              -34.61997
            ],
            [
              -58.34633,
              -34.61921
            ],
            [
              -58.34519,
              -34.61983
            ],
            [
              -58.3446,
              -34.61899
            ],
            [
              -58.34782,
              -34.61895
            ],
            [
              -58.35102,
              -34.61978
            ],
            [
              -58.35142,
              -34.61893
            ],
            [
              -58.34978,
              -34.61714
            ],
            [
              -58.34481,
              -34.61772
            ],
            [
              -58.34088,
              -34.61697
            ],
            [
              -58.33979,
              -34.61604
            ],
            [
              -58.34082,
              -34.6107
            ],
            [
              -58.34573,
              -34.60687
            ],
            [
              -58.34728,
              -34.60169
            ],
            [
              -58.3555,
              -34.59524
            ],
            [
              -58.3571,
              -34.59483
            ],
            [
              -58.36007,
              -34.59498
            ],
            [
              -58.36341,
              -34.59666
            ],
            [
              -58.36436,
              -34.59537
            ],
            [
              -58.3638,
              -34.59719
            ],
            [
              -58.3679,
              -34.5975
            ],
            [
              -58.36908,
              -34.59414
            ],
            [
              -58.36636,
              -34.59296
            ],
            [
              -58.3648,
              -34.59447
            ],
            [
              -58.36436,
              -34.59427
            ],
            [
              -58.36651,
              -34.59096
            ],
            [
              -58.36789,
              -34.59001
            ],
            [
              -58.35945,
              -34.58809
            ],
            [
              -58.35959,
              -34.58717
            ],
            [
              -58.36351,
              -34.58747
            ],
            [
              -58.36368,
              -34.58624
            ],
            [
              -58.35965,
              -34.58591
            ],
            [
              -58.35987,
              -34.58409
            ],
            [
              -58.365,
              -34.5845
            ],
            [
              -58.36615,
              -34.58354
            ],
            [
              -58.36142,
              -34.58296
            ],
            [
              -58.36163,
              -34.58113
            ],
            [
              -58.36898,
              -34.58171
            ],
            [
              -58.37061,
              -34.58058
            ],
            [
              -58.36526,
              -34.58015
            ],
            [
              -58.36529,
              -34.57971
            ],
            [
              -58.36794,
              -34.57819
            ],
            [
              -58.3743,
              -34.57869
            ],
            [
              -58.37604,
              -34.57747
            ],
            [
              -58.37073,
              -34.57705
            ],
            [
              -58.37074,
              -34.57659
            ],
            [
              -58.37548,
              -34.57389
            ],
            [
              -58.38184,
              -34.57439
            ],
            [
              -58.38359,
              -34.5734
            ],
            [
              -58.37772,
              -34.57269
            ],
            [
              -58.38113,
              -34.57062
            ],
            [
              -58.37877,
              -34.57042
            ],
            [
              -58.37628,
              -34.57179
            ],
            [
              -58.37129,
              -34.57146
            ],
            [
              -58.37549,
              -34.56792
            ],
            [
              -58.39049,
              -34.5694
            ],
            [
              -58.3912,
              -34.57058
            ],
            [
              -58.38664,
              -34.57323
            ],
            [
              -58.38795,
              -34.57482
            ],
            [
              -58.39301,
              -34.57196
            ],
            [
              -58.39646,
              -34.56689
            ],
            [
              -58.39732,
              -34.56666
            ],
            [
              -58.39695,
              -34.56587
            ],
            [
              -58.39695,
              -34.56515
            ],
            [
              -58.39923,
              -34.56371
            ],
            [
              -58.4007,
              -34.5643
            ],
            [
              -58.40105,
              -34.56335
            ],
            [
              -58.40274,
              -34.56395
            ],
            [
              -58.40441,
              -34.56283
            ],
            [
              -58.40927,
              -34.55846
            ],
            [
              -58.40946,
              -34.5567
            ],
            [
              -58.41252,
              -34.55514
            ],
            [
              -58.42651,
              -34.55182
            ],
            [
              -58.42554,
              -34.54885
            ],
            [
              -58.42707,
              -34.54807
            ],
            [
              -58.42977,
              -34.5486
            ],
            [
              -58.43074,
              -34.5478
            ],
            [
              -58.4313,
              -34.5448
            ],
            [
              -58.43456,
              -34.54263
            ],
            [
              -58.43561,
              -34.54084
            ],
            [
              -58.4377,
              -34.53996
            ],
            [
              -58.4403,
              -34.53985
            ],
            [
              -58.44077,
              -34.54034
            ],
            [
              -58.4415,
              -34.53785
            ],
            [
              -58.44361,
              -34.5363
            ],
            [
              -58.44623,
              -34.53593
            ],
            [
              -58.44856,
              -34.5374
            ],
            [
              -58.44937,
              -34.53633
            ],
            [
              -58.44904,
              -34.53554
            ],
            [
              -58.45002,
              -34.53607
            ],
            [
              -58.44987,
              -34.53736
            ],
            [
              -58.45219,
              -34.53964
            ],
            [
              -58.45381,
              -34.53898
            ],
            [
              -58.45499,
              -34.53738
            ],
            [
              -58.453,
              -34.5357
            ],
            [
              -58.45036,
              -34.53556
            ],
            [
              -58.45011,
              -34.53437
            ],
            [
              -58.45089,
              -34.53205
            ],
            [
              -58.45213,
              -34.53132
            ],
            [
              -58.4518,
              -34.53013
            ],
            [
              -58.45401,
              -34.52931
            ],
            [
              -58.45688,
              -34.52667
            ],
            [
              -58.45915,
              -34.527
            ],
            [
              -58.4615,
              -34.53131
            ],
            [
              -58.46384,
              -34.53326
            ],
            [
              -58.46693,
              -34.53468
            ]
          ]
        ]
    }
```

**Pull request merged**